cumulus/parachains/runtimes/collectives/collectives-westend/Cargo.toml

```
@@ -0,0 +1,226 @@
+[package]
+name = "collectives-westend-runtime"
+version = "1.0.0"
+authors.workspace = true
+edition.workspace = true
+license = "Apache-2.0"
+description = "Westend Collectives Parachain Runtime"
+
```

```suggestion
publish = false

```
not published to crates.io

---

```
collectives-westend-runtime (cumulus/parachains/runtimes/collectives/collectives-westend):
    "publish = false" is set but this crate is a dependency of others

glutton-westend-runtime (cumulus/parachains/runtimes/glutton/glutton-westend):
    "publish = false" is set but this crate is a dependency of others
```
For now these can't be unpublished because they're used by `polkadot-parachain-bin`. `crates.io` requires that every dependency required by a published package is also published.